susemanager-utils/testing/automation/obs-project.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/python3
+import argparse
+import os
+import sys
+import urllib.request
+import xml.etree.ElementTree as ET
+import datetime
+import configparser
+
+
+
+def add(args):
+    api = args.api
+    project = args.project
+    pr_project = args.prproject
+    pull_number = args.pullnumber
+    maintainer=args.setmaintainer
+    pr_project = pr_project + ":" + pull_number
+    config_file = args.configfile
+
+    if (not os.path.exists(config_file)):
+        print("ERROR: config file {} not found".format(config_file))
+        sys.exit(-1)
+
+    config = configparser.ConfigParser()
+    try:
+        config.read(config_file)
+    except IOError as e:
+        print("ERROR: Can't read config file ".format(e))
+        sys.exit(-1)
+
+    auth_user = config[api]["user"]
```

Comment:
You can also use `osc` to read the config (if installed, of course), it also works for non-plaintext passwords. We use that for a small script that reads the Salt versions from specfiles, added in https://github.com/brejoc/saltversions/commit/417deb7f7dcabe3b79b01dc96910ba7b391aac13